Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -53,6 +53,7 @@ async function getKubeConfigFile(): Promise<string> {
         var configFilePath = getKubeConfigFilePath();
         tl.debug(tl.loc("KubeConfigFilePath", configFilePath));
         fs.writeFileSync(configFilePath, config);
+        fs.chmodSync(configFilePath, 0o700);
```

Comment:
Can we check what are the current permissions that the file has? We can then choose to keep the user part of those permission and:
- Test the helm commands.
- Test on a self-hosted agent too.

Then we can confidently take this change.

Thoughts? @shigupt202 @ammohant

Reply:
yes, we should check the current persm to be safer

Reply:
Checked the permissions. The user permissions are read and write only, not execute. I think we can keep it the same for user level permissions, and remove all group and world level permissions.

Reply:
Makes sense. I don't think it is used anywhere locally. Makes sense then. Let us keep read and write perms for the user in that case and take away the grp and others.

The helm tool complains about the write and execute or even for read perms for others.

Reply:
The warning is for read perms on group and world level.
`WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/vsts/work/_temp/helmTask/1601032829368/config`
`WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/vsts/work/_temp/helmTask/1601032829368/config`

Reply:
ok, Iam good with the change. Just make sure you test this well.